client/pool.c

```diff
@@ -772,6 +788,30 @@ static int is_block_data_received(connection_list_element *connection)
 				}
 
 				pthread_mutex_unlock(&g_pool_mutex);
+
+				if(g_multi_address) {
```

Comment:
What will happen if wallet is multi addresses connect to pool without activated multi addresses? And vice versa.

Reply:
What will happen if transaction will be sent?

Reply:
If pool does not enable multi addresses, this package will be no processed.